app/src/behaviors/behavior_bt.c

```diff
@@ -20,6 +20,73 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 #if DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT)
 
+#if IS_ENABLED(CONFIG_ZMK_BEHAVIOR_METADATA)
+
+static const struct behavior_parameter_value_metadata no_arg_values[] = {
+    {
+        .position = 0,
+        .value = BT_NXT_CMD,
+        .friendly_name = "Next Profile",
+        .type = BEHAVIOR_PARAMETER_VALUE_METADATA_TYPE_VALUE,
```

Comment:
Nit: I think it would be easier to read these if the type came before the value/range/etc. Maybe this would be a good order?

1. Position
2. Name
3. Type
4. Value/range/etc

Reply:
Fixed. Definitely reads better, thanks!